omnigibson/robots/fetch.py

```diff
@@ -471,12 +471,16 @@ def urdf_path(self):
 
     @property
     def arm_workspace_range(self):
-        return {self.default_arm: [th.deg2rad(th.tensor([-45])).item(), th.deg2rad(th.tensor([45])).item()]}
+        return {
+            self.default_arm: th.tensor(
+                [th.deg2rad(th.tensor([-45])).item(), th.deg2rad(th.tensor([45])).item()], dtype=th.float32
+            )
```

Comment:
I think looking at this line you can tell there _must_ be an easier way! Simply: `self.default_arm = th.deg2rad(th.tensor([-45, 45], dtype=th.float32))`